Frameworks/CoreText/CTFrame.mm

```diff
@@ -111,16 +111,23 @@ void CTFrameGetLineOrigins(CTFrameRef frameRef, CFRange range, CGPoint origins[]
 /**
  @Status Interoperable
 */
-void CTFrameDraw(CTFrameRef frame, CGContextRef ctx) {
-    uint32_t count = [((_CTFrame*)frame)->_lines count];
-    CGPoint curTextPos = CGContextGetTextPosition(ctx);
+void CTFrameDraw(CTFrameRef frameRef, CGContextRef ctx) {
+    _CTFrame* frame = static_cast<_CTFrame*>(frameRef);
```

Comment:
again use the macros, it can be done before the cast, and it makes the code much more readable.